zoom.sh

```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/env bash
 
+CONF_FILE=$HOME/.zoom/config/zoomus.conf
+
 # Check if the current desktop is using Wayland
 if [[ "$XDG_SESSION_TYPE" == "wayland" ]]; then
```

Comment:
```suggestion
if [ -z "$DISPLAY" ] && [ -n "$WAYLAND_DISPLAY" ]; then
```

True if actively running in Wayland and Wayland Flatpak socket is enabled. Together with the above change (don't pass Wayland socket by default) means Zoom will run in Xwayland by default, and if someone enables the Wayland Flatpak socket it will run in Native Wayland mode.

Inspired by https://github.com/flathub/com.slack.Slack/pull/139